src/StakingNode.sol

```diff
@@ -257,23 +280,131 @@ contract StakingNode is IStakingNode, StakingNodeEvents, ReentrancyGuardUpgradea
         }
     }
 
+    /**
+     * @notice Queues multiple withdrawals for processing.
+     */
+    function queueWithdrawals(
+        uint256 sharesAmount
+    ) external onlyOperator returns (bytes32[] memory fullWithdrawalRoots) {
+
+        IDelegationManager delegationManager = IDelegationManager(address(stakingNodesManager.delegationManager()));
```

Comment:
Redundant casts. can simply call `stakingNodesManager.delegationManager()`. Also redundant state variable though, used only once.

Reply:
yep, lets remove the above line and replace line 367 with:

```solidity
fullWithdrawalRoots = stakingNodesManager.delegationManager().queueWithdrawals(params);
```